pkg/prom/instance/instance.go

```diff
@@ -44,7 +44,7 @@ var (
 var (
 	DefaultConfig = Config{
 		HostFilter:           false,
-		WALTruncateFrequency: 1 * time.Minute,
+		WALTruncateFrequency: 60 * time.Minute,
```

Comment:
## Some thoughts

Switching the frequency from 1m to 60m it means in agent with low/moderate traffic you will have few segments over the last 60m.

Before this change you were creating a segment every 1m, while after this change a new segment is created once the previous one hit 128MB size or after 60m.

The `Storage.Truncate()` creates a checkpoint if there are at least 3 segments and the checkpoint will contain the low 1/3 of segments. Let's consider two opposite scenarios:
- High volume (1 segment / minute): every 60m we create a checkpoint. The checkpoint contains the oldest 20m and the WAL segments contain the newest 40m. The longest outage we can tolerate is 40m.
- Low volume (1 segment / hour): every 60m we create a checkpoint. Since we have 1 segment / hour, the checkpoint contains the oldest 1h and the WAL segments contain the newest 2h. The longest outage we can tolerate is 2h.

@rfratto @codesome is my analysis ☝️  correct? I'm wondering if:
1. the agent should reduce the max segment size or make it configurable
2. we should document the longest outage without data loss is two third of `wal_truncate_frequency`